roles/nova-data/tasks/libvirt.yml

```diff
@@ -3,13 +3,35 @@
 - name: enable nested kvm
   template: src=etc/modprobe.d/kvm-nested.conf dest=/etc/modprobe.d/kvm-nested.conf owner=root group=root mode=0644
 
+- name: Install key for Ubuntu Cloud Archive
+  apt: pkg=ubuntu-cloud-keyring update_cache=yes
+  when: ansible_distribution_version == "12.04"
```

Comment:
This needs to be tagged security too, otherwise we may not have the GPG key to validate the new QEMU KVM package.